CHANGES.rst

```diff
@@ -90,6 +90,12 @@ astropy.time
 astropy.units
```

Comment:
Wrong section 😄

Reply:
Oops, good catch, thanks!
I moved the two sentences to the right section and skipped the CI as it passed previously.